client/go/commands/cluster_destroy.go

```diff
@@ -96,8 +96,13 @@ func (a *DestroyClusterCommand) Exec(args []string) error {
 	}
 
 	//if all is well, print stuff
-	fmt.Fprintf(stdout, "Successfully destroyed cluster with id: %v ", clusterId)
+	if a.options.Json == false {
```

Comment:
Since this command does not return JSON, do not print anything into  the stdout, because it would mess up parsers.  Instead just return success or failure.

Therefore you should change the `if` to:

```
if !a.options.Json {
   ... Print strings ...
}
```